Assets/HotmaxEditorVR/handColliderFollow.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class handColliderFollow : MonoBehaviour
+{
+    //--local refs
+    private stateManager _stateManagerMutatorRef;
+    private bool _selectedObjectIsActive = stateManager.selectedObjectIsActive;
+
+    private GameObject collidedWithHand;
+
+    Transform selectorHand;
+
+    private bool isCurrentlyColliding = false;
+
+    private void Awake()
+    {
+        _stateManagerMutatorRef = GameObject.FindObjectOfType(typeof(stateManager)) as stateManager;
+
+        stateManager.selectedObjectIsActiveEvent += updateSelectedObjectIsActive;
+
+        selectorHand = GameObject.Find("Hand2").transform;
+    }
+
+    private void OnApplicationQuit()
+    {
+        stateManager.selectedObjectIsActiveEvent -= updateSelectedObjectIsActive;
+    }
+
+    void updateSelectedObjectIsActive(bool value)
+    {
+        _selectedObjectIsActive = value;
+    }
+
+    void Update()
+    {
+        transform.position = selectorHand.position;
+        transform.rotation = selectorHand.rotation;
+    }
+
+    private void OnTriggerEnter(Collider other)
+    {
+        if (other.gameObject.GetComponent<MonoBehaviour>() is prop)
+        {
+            if(!isCurrentlyColliding && !_selectedObjectIsActive)
```

Comment:
add above nest if `//TODO: if you bring object in from telekinesis mode, and unclick, you have to still leave and enter object to free grab'

Reply:
got it